src/partials/pages/designer-stories/_hero-fadzly-sayarasi.html

```diff
@@ -0,0 +1,24 @@
+<section class="hero dms-band pt-0">
+    <div class="row">
+        <div class="col-12 col-xl-10 offset-xl-3 px-0">
+            <img class="img-fluid hero-img" src="/assets/img/pages/designer-stories/fadzly-sayarasi/img-hero.png" alt="Allona's desk">
```

Comment:
The `alt` text here doesn't seem correct.

Reply:
Updated.